### PR TITLE
Change name of MTT to MPT, mttp to mmpt, & ext. name from Smmtt to Smmpt

### PR DESCRIPTION
**Pull request by rsahita**
Address issue #91

### CODE REVIEW

File: chapter6.adoc [hunk 1]
```diff
@@ -66,51 +66,51 @@ response to the address translation request on the device translation completion
...
-MMU allows the same table to be used simultaneously by the CPU MMU and MTTCHK.
-The MTT access permission lookup process used by MTTCHK is identical to that
-specified by Smmtt extension in section "MTT access permissions lookup process".
+the MPT check completion (MCC) interface to the IO Bridge. As part of the MPT
+check, the MPTCHK may need to perform implicit accesses to the MPT using the MPT
+walk interface (MWI). To perform the checks the MPTCHK uses the same MPT table
+format as used by the CPU's MMU. Using the same MPT table formats as the CPU's
+MMU allows the same table to be used simultaneously by the CPU MMU and MPTCHK.
+The MPT access permission lookup process used by MPTCHK is identical to that
+specified by Smmpt extension in section "MPT access permissions lookup process".
```
Comment (eckhard-delfs-qualcomm) [inline chapter6.adoc:80]:
```suggestion
specified by `Smmpt` extension in section "MPT access permissions lookup process".
```

File: chapter6.adoc [hunk 2]
```diff
@@ -10,49 +10,49 @@ and protection for DMA that originates from devices under its control.
...
 
 * Supervisor Domain Classifier (SDCL): This classifier within the I/O subsystem
-  interprets the attributes of a DMA request and determines the appropriate MTT
+  interprets the attributes of a DMA request and determines the appropriate MPT
   for that request.
 
-* MTT Checker (MTTCHK): This function ensures that stipulated access controls by
-  Smmtt are applied to the memory regions accessed by the DMA. It uses the MTT
+* MPT Checker (MPTCHK): This function ensures that stipulated access controls by
+  Smmpt are applied to the memory regions accessed by the DMA. It uses the MPT
```
Comment (eckhard-delfs-qualcomm) [inline chapter6.adoc:36]:
```suggestion
  `Smmpt` are applied to the memory regions accessed by the DMA. It uses the MPT
```

**Review by eckhard-delfs-qualcomm — COMMENTED** [review]
There are 2 `MTT` occurrences in chapter 8.4.
Furthermore, figures 1,2,7,8,14 need some MTT/mttp related patch.

**Comment by rsahita** [comment]
> There are 2 `MTT` occurrences in chapter 8.4. Furthermore, figures 1,2,7,8,14 need some MTT/mttp related patch.

yes figures need to be updated (was knocking out the text changes first)

**Review by eckhard-delfs-qualcomm — COMMENTED** [review]
Looks good.

Just 2 missing MTT references in chapter 8 (section 8.4). 

_Capacity and bandwidth controllers that support supervisor domains provide one or more memory-mapped QoS register interfaces (QRI). A QRI may be made exclusively accessible to a supervisor domain or the RDSM may mediate access to the QRI using an SBI. The RDSM can control access to the QRI from supervisor domains using **MTT** and/or PMP. The RDSM controls access to the QRI from devices using IO**MTT** and/or IOPMP_

**Comment by rsahita** [comment]
Figures also updated

**Pull request merged**